composer.json

```diff
@@ -55,18 +55,19 @@
         "symfony/polyfill-php56": "*"
     },
     "scripts": {
-        "auto-scripts": [
+        "sulu-auto-scripts": [
+            "echo 'Use sulu-auto-scripts to avoid problems with symfony/flex: https://github.com/symfony/flex/issues/431'",
```

Comment:
Do we need this echo? Doesn't that only clutter the output?

Reply:
@danrot did remove it. first I thought I would change it to a bash command `# ...` but think this could crash on other platforms like windows.

Reply:
This, and I think it doesn't really matter anyway how these scripts are called :-)